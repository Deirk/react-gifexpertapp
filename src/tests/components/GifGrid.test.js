import React from 'react';
import { shallow } from "enzyme"
import { GifGrid } from "../../components/GifGrid"

describe('Pruebas en el archivo GifGrid.js', () => {
    
    const category = 'Mia';
    const wrapper = shallow(<GifGrid category={category}/>);

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})
