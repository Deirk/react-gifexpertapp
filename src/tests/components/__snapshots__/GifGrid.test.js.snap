// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el archivo GifGrid.js debe mostrarse correctamente 1`] = `
<Fragment>
  <h3
    className="animate__fadeIn"
  >
    Mia
  </h3>
  <div
    className="card-grid"
  >
    <p
      className="animate__flash"
    >
      Loading...
    </p>
  </div>
</Fragment>
`;
